### PR TITLE
Add links to additional foundational libraries

### PR DESCRIPTION
**Pull request by AxiomaticSemantics**
# Objective

- Add links to various foundational libraries

## Solution

- Add the links

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -86,13 +86,19 @@ fn main(){
...
+Bevy as a whole is only possible because of the hard work put into these foundational technologies:
 
-* [wgpu](https://wgpu.rs/): modern / low-level / cross-platform graphics library based on the [WebGPU](https://gpuweb.github.io/gpuweb/) API.
+* [ab-glyph](https://github.com/alexheretic/ab-glyph): font loading, scaling, positioning and rasterization
+* [accesskit](https://github.com/AccessKit/accesskit): cross-platform accessibility(ally) support
+* [gilrs](https://gitlab.com/gilrs-project/gilrs): cross-platform game input device handling
 * [glam-rs](https://github.com/bitshifter/glam-rs): a simple and fast 3D math library for games and graphics
+* [gltf](https://github.com/gltf-rs/gltf): glTF 2.0 loader
+* [rodio](https://github.com/RustAudio/rodio): audio playback library leveraging [CPAL](https://github.com/RustAudio/cpal)
+* [taffy](https://github.com/DioxusLabs/taffy): cross-platform UI layout library
```
Comment (JMS55) [inline README.md:99]: Correct me if I'm wrong, but taffy doesn't seem to depend on anything platform specific? Sure it's cross platform, but that's not really saying anything considering it's not abstracting over different platforms in the first place (like wgpu or winit does) xD.

Anyways small nit, lgtm.

**Review by JMS55 — APPROVED** [review]

**Comment by mockersf** [comment]
I would prefer to remove that part of the readme rather than to extend it with more libraries.

Some of those are not foundational for me, and there are active efforts to remove them

**Comment by cart** [comment]
Closing in favor of a PR that removes it.

**Comment by cart** [comment]
https://github.com/bevyengine/bevy/pull/10836

**Pull request closed without merging**